src/Illuminate/Routing/Controller.php

```diff
@@ -54,6 +54,20 @@ public function callAction($method, $parameters)
         return call_user_func_array([$this, $method], $parameters);
     }
 
+    /**
+     * Return a redirect to a method in this Controller.
+     *
+     * @param $method
+     * @param array $parameters
+     * @param int $status
+     * @param array $headers
+     * @return \Illuminate\Http\RedirectResponse
```

Comment:
Can you format these params like we do in the rest of the framework? Also add a type-hint for the `$method` param.